Libraries/D3D12RaytracingFallback/src/CalculateMortonCodesForAABBs.hlsl

```diff
@@ -26,3 +26,5 @@ float3 GetCentroid(uint elementIndex)
 {
     return ReadBottomLevelBoundingBox(elementIndex).center;
 }
+
+#include "CalculateMortonCodes.hlsli"
```

Comment:
The includes should be with the rest of the includes at the top of the file.

Reply:
I'm following a pattern (seen in the [top-level compute AABB pass](https://github.com/Microsoft/DirectX-Graphics-Samples/blob/develop-dxr-rs5/Libraries/D3D12RaytracingFallback/src/TopLevelComputeAABBs.hlsl), for example) where the shader is defining helper functions that the [main logic will eventually use](https://github.com/Microsoft/DirectX-Graphics-Samples/pull/435/files/eec10f868bd796311c7ad3fab2dc0a710552cef8#diff-5d48fb565cbf48d0c7d782e5ec8eac59R114) "below" it.

Reply:
I find it weird, but I'm OK with if it's already a trend.

Reply:
+1 this pattern is weird